docs/epiviz/overview.rst

```diff
@@ -0,0 +1,76 @@
+.. _overview-of-epiviz:
+
+Overview of EpiViz Runner
+=========================
+
+.. _epiviz-overview-introduction:
+
+Introduction
+------------
+EpiViz-AT is a web page on http://epimodeling-web-d01.ihme.washington.edu/at
+or http://epimodeling-web-p01.ihme.washington.edu/at. After you fill out
+that web page and hit submit, the web server runs a program on the cluster.
+We call that program ``epiviz_runner``. It is installed on the IHME cluster,
+and the code itself is stored at https://github.com/ihmeuw/cascade.
+The ``epiviz_runner``, itself, is at
+https://github.com/ihmeuw/cascade/blob/develop/src/cascade/executor/epiviz_runner.py.
```

Comment:
I'm questioning this whole section of docs, really. I needed to write this in order to get my head straight on how to do study covariates. For instance, I knew the bundle had them but forgot that all of the measurements, including the mtall that we add to it, need to have these same study covariates. I do want one reference so that there is a breadcrumb. Maybe I can make it a "ref" though...